src/components/GeneralInfo/LoadingCard/LoadingCard.test.js

```diff
@@ -1,8 +1,10 @@
+import '@testing-library/jest-dom';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import React from 'react';
-import toJson from 'enzyme-to-json';
+import { TestWrapper } from '../../../Utilities/TestingUtilities';
```

Comment:
I'm feeling a bit odd about `renderWithRouter` vs. `TestWrapper`. Both basically accomplish the same thing, but there is no real pattern to when to use which.

Maybe we can utilise one or the other and that consistently throughout to avoid confusion? 

Personally, I'd be more in favour of `TestWrapper` as it also provides other things. 
I'm also a bit iffy about wrapping `render`.